src/theme.js

```diff
@@ -0,0 +1,36 @@
+import { createTheme } from '@mui/material/styles';
+
+const theme = createTheme({
+  typography: {
+    fontFamily: ['Roboto', 'sans-serif'].join(',')
```

Comment:
If we really just want a fallback of "sans-serif" instead of the browser's default, in case "Roboto" breaks for any reason, I think we can just use fontFamily: "Roboto, san-serif".

This seems overly complicated? I think MUI's documentation also allows the syntax that I've just used for fontFamily. (see theme for fontFamily specifically)

![Screen Shot 2023-05-05 at 19 51 23](https://user-images.githubusercontent.com/14917816/236594804-73ea21e9-80b2-4517-bba6-69116bc3d6cf.png)